diagnostics/model_test/README.md

```diff
@@ -0,0 +1,21 @@
+# Compare model output between Tensorflow and NGraph
+
+### This model_test tool will run the model inference seperately on TF and NGraph, and the desired output from TF and NGraph should match given the same inputs. It can be used as a debugging tool, and also a verification that NGraph produces the same output as Tensorflow. 
+
+# Required files to use the tool:
+* A json file: Provide model specific parameters. Follow the same format as the ```mnist_cnn.json`` 
+* A tensorflow frozen graph: A model frozen graph(.pb file) with trained weights and model architecture
+
+## To prepare the required json file:"
+* You will need the names of the input/output tensors of the model. Currently we are supporting
+multiple input tensors and one output tensor. Put the input tensor names as a list in the ```input_tensor_name``` field of the json file, and the output tensor name as a string in the ```output_tensor_name``` field of the json file
+* You will need the input dimensions for all the input tensors provided. Put the dimensions information as a list in the ```input_dimension``` field of the json file, and the corresponding order of ```input_tensor_name``` list should match the ```input_dimension``` list. Therfore, the length of ```input_tensor_name``` list should match the length of ```input_dimension``` list
+* Specify the the location of the frozen graph in the ```frozen_graph_location``` of the json file
+* Specify the ```batch_size``` field in the json file to the desired batch size for inference
+* Specify the tolerance between the TF and NGraph outputs at ```l1_norm_threshold```, ```l2_norm_threshold``` and ```inf_norm_threshold``` in the json file 
+
+# To run the model test tool:
+	python verify_model.py --json_file="./mnist_cnn.json" 
```

Comment:
May you can change this to python verify_model.py --json_file="<path to json file>"

Reply:
I feel we should not have a default for ```--json_file``` option, as it is _the_ key input to the code.
In any case, the current mnist.json will not work for an external user (since the .pb in the son is internal).

Therefore without a default, we would force users to supply their own .json so that they do not use our .json by mistake and find that it fails.